### ISSUE TITLE
Path to .env file in nuxt.config

### ISSUE THREAD

**Issue by fomalhaut79** [wemake-services/wemake-vue-template#640]
Path to .env file in nuxt.config.js: `['@nuxtjs/dotenv', { path: envPath }]` should be `path.resolve(__dirname, 'config')` instead of `path.resolve(__dirname, 'config', '.env')` when used for dotenv-module

**Comment by sobolevn** [comment]
Indeed, this line tells us that you are right: https://github.com/nuxt-community/dotenv-module/blob/master/src/index.js#L13

@fomalhaut79 can you please send a PR with the fix?
